design/AstroMancer Cards.md

```diff
@@ -0,0 +1,21 @@
+##AstroMancer Cards
+
+###AstroMancer Cards can be played for 2 Mana
+
+Starfall -2 Mana Quadra-cast(Total cost = 2) Deal 6 damage to a random enemy. Feed: Heal a random ally 2.(So 8 healing in all.)
+
+Star Slash +1 Mana deal .5 damage to an enemy. Feed: Heal 1 for each mana.
+
+Starlight -1 Mana Deal 5 damage to all enemies. Feed: The Solar card has +10% spell Power for each enemy hit.
```

Comment:
What is `spell Power`? 

Is an enemy hit a pre-calculation? Is it boosted based on the number of enemies targetted, before attacking? Or based on the number of enemies previous attacked by this charater during this turn? Or something else?